crates/core/src/host/wasm_common.rs

```diff
@@ -359,7 +359,6 @@ macro_rules! abi_funcs {
             "spacetime_8.0"::buffer_alloc,
             "spacetime_8.0"::buffer_consume,
             "spacetime_8.0"::buffer_len,
```

Comment:
What semantic do we use here? should this be updated to 9.0 now?

Reply:
This is a breaking change and should probably be set to 9.0

Reply:
yeah, I have already done that